examples/pylab_examples/fonts_demo.py

```diff
@@ -13,7 +13,7 @@
 alignment = {'horizontalalignment': 'center', 'verticalalignment': 'baseline'}
 # Show family options
 
-families = ['serif', 'sans-serif', 'cursive', 'fantasy', 'monospace']
+families = ['serif', 'sans-serif', 'monospace']
```

Comment:
I spent some time at some point finding fonts for all the categories that are free and installed on Travis. See the .travis file. Before that it was a bit of a mess